extension/src/pipeline/quickPick.ts

```diff
@@ -8,18 +8,14 @@ import {
   isValueTree
 } from '../cli/dvc/contract'
 import { getFileExtension, loadDataFiles } from '../fileSystem'
-import {
-  quickPickManyValues,
-  quickPickOne,
-  quickPickValue
-} from '../vscode/quickPick'
+import { quickPickOne, quickPickUserOrderedValues } from '../vscode/quickPick'
 import { pickFiles } from '../vscode/resourcePicker'
 import { Title } from '../vscode/title'
 import { Toast } from '../vscode/toast'
 import { getInput } from '../vscode/inputBox'
 
 export type PlotConfigData = {
-  x: { [file: string]: string }
+  x: { [file: string]: string[] | string }
```

Comment:
[Q] Why do we have a mixed data type here? Why not always have a string array?

Reply:
I chose to have a mixed data type so we could need to loop over `x`/`y` fields twice, one over here and again in the yaml plot obj. But I see that it would simplify things if we just looped over it twice. I'll update the code to just use a string array.